config.toml

```diff
@@ -27,4 +27,4 @@ lang = "en"
 github = "reside-ic"
 posts_navigation = true
 small_banner_logo = false
-latestpostcount = 5
\ No newline at end of file
+latestpostcount = 5
```

Comment:
not sure what I did here

Reply:
should I remove it?

Reply:
no, it's totally fine (my editor adds them too)